src/types/structures.ts

```diff
@@ -113,14 +118,16 @@ export interface zDAOProperties {
   zNAs: zNA[]; // Linked zNA
   title: string; // zDAO title, zNA by default
   creator: string; // Creator wallet address
-  // avatar uri which starts with https schema
-  // The frontend should use default avatar image if not defined
-  avatar?: string; // Avatar uri (https link)
   network: string; // Chain id
   duration?: number; // Proposal duration if DAO has fixed duration
   safeAddress: string; // Gnosis Safe address
   votingToken: Token; // Voting token
+  amount: string; // Minimum amount of voting power to cast a vote
```

Comment:
This comment is incorrect. What we want is the minimum amount of voting power for a user to **create a proposal** not cast a vote.